docs/source/metoffice.rst

```diff
@@ -0,0 +1,165 @@
+.. _MetOffice:
+
+Using Fab at the Met Office
+===========================
+
+For users working on Met Office desktop systems the following instructions may
+prove useful.
+
+Using Fab
+~~~~~~~~~
+
+If you just want to use Fab to build some source then there are a number of
```

Comment:
I wasn't too sure about this sentence – might this be a typo?

```suggestion
If you just want to use Fab to build from source then there are a number of
```

Reply:
Tried an alternative.